package Exception_Tasks;

public class Array_Out_Of_bound {


    public static void main(String[] args) {
        int [] Arr= {1,2,4,5,6,7};
        for(int i=0;i<=Arr.length;i++){
            System.out.println(Arr[i]);
        }
    }
}
